swiss-ng-api/berlin-group-documentation/current.yaml

```diff
@@ -8231,6 +8419,38 @@ components:
       maxLength: 500
 
 
+    tppMessageGeneric: 
+      type: object
+      required:
+        - category
+        - code
+      properties:
+        category:
+           $ref: "#/components/schemas/tppMessageCategory"
+        code:
+          $ref: "#/components/schemas/tppMessageCategory"
+        path:
+          type: string
+        text:
+          $ref: "#/components/schemas/tppMessageText"
+
+
+    tppMessageInitiationStatusResponse-202: 
+      type: object
+      required:
+        - category
+        - code
+      properties:
+        category:
+           $ref: "#/components/schemas/tppMessageCategory"
+        code:
+          $ref: "#/components/schemas/MessageCode202InitiationStatus"
```

Comment:
Relevant Change: MessageCode202InitiationStatus is new.
Adoption approach: Follow the change